chainercv/transforms/bbox/crop_bbox.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+
+
+def crop_bbox(
+        bbox, y_slice=None, x_slice=None,
+        allow_outside_center=True, return_param=False):
+    """Crop bounding boxes.
```

Comment:
Yeah, but main purpose of this function is not to crop bounding boxes into smaller pieces.
Its main purpose is to adjust bounding boxes to reflect image cropping.

I think this should be clear in the beginning of the doc.